resume.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Ansible playbook to resume Slurm compute nodes. Waits for compute nodes to
+# change state for 5 minutes.
+#
+# Variables:
+# - nodes_to_resume: list of compute nodes to resume
+
+- name: Resume compute nodes
```

Comment:
For idempotency, is there a means of checking the state before we change it to `RESUME`

Reply:
I am not sure I understand how it relates to idempotence. Did you mean that, for nodes that are already in DRAIN state (which could have been done by an admin), we should not resume them?